Documentation/ceph-csi-drivers.md

```diff
@@ -138,3 +138,24 @@ csi_liveness 1
 
 Check the [monitoring doc](ceph-monitoring.md) to see how to integrate CSI
 liveness and grpc metrics into ceph monitoring.
+
+### Dynamically Expand Volume
```

Comment:
Two suggestions:
- Make this a level 2 header
- This comment at the top is out of date: "The remainder of this topic is on the snapshotting feature of the RBD driver." How about changing it to mention resizing is also available

Reply:
removed Remaining topic in doc